scripts/create_config.py

```diff
@@ -10,17 +10,22 @@
 parser = argparse.ArgumentParser(description='Create config from gmsl and gmst pulse files')
 
 # Add named arguments for the lists
-parser.add_argument('--gmsl_file', nargs=1, help='Path to GMSL pulse file')
-parser.add_argument('--gmst_file', nargs=1, help='Path to GMST pulse file')
+parser.add_argument('--gmsl_file', type=str, help='Path to GMSL pulse file')
```

Comment:
Might want to change help message to "Full path for GMSL pulse file, including filename" or something like that (same for GMST file).